connector_dns/dns.py

```diff
@@ -0,0 +1,112 @@
+# -*- coding: utf-8 -*-
+# Copyright 2015 Elico Corp
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+from openerp import models, fields, api
+
+
+class DNSBackend(models.Model):
+    _name = 'dns.backend'
+    _inherit = 'connector.backend'
+    _backend_type = 'dns'
+
+    def _select_version(self):
+        return []
+
+    login = fields.Char(
+        string='Login',
+        help="Provider's login.",
+        required=True
+    )
+    password = fields.Char(
+        string='Password',
+        help="Provider's password.",
+        required=True
+    )
+    state = fields.Selection(
+        [('draft', 'Draft'), ('done', 'Done'),
+         ('exception', 'Exception')],
+        string='State',
+        default="draft",
+        help='"Confirmed" when the domain has been succesfully created.'
+    )
+    version = fields.Selection(
+        selection='_select_version',
+        string='Service Provider',
+        help='DNS service provider',
+        required=True
+    )
+
+    @api.multi
+    def name_get(self):
+        res = []
+        for backend in self:
+            res.append((backend.id, '%s (%s)' % (backend.name, backend.login)))
+        return res
+
+
+class DNSDomain(models.Model):
+    _name = 'dns.domain'
+    _inherit = 'dns.binding'
+
+    name = fields.Char(
+        string='Name',
+        required=True,
+        help='Domain name without "www", such as"dnspod.cn"'
+    )
+    record_ids = fields.One2many(
+        comodel_name='dns.record',
+        inverse_name='domain_id',
+        string='Subdomains'
+    )
```

Comment:
Typically the bind records are `_inherits` from regular Odoo records. I get that the regular record doesn't exist, but I think this is also breaking some core logic & not following the [interface spec](https://github.com/OCA/connector/blob/7dfc607ae31fbb2a0a4da5c3a4979401e75d185c/connector/backend_model.py#L69).

Note it says `openerp_id` in the spec, but I have been using `odoo_id` for new connectors. That's a design choice, but IMO it's best to deprecate anything named `openerp` for `odoo`